spec/concerns/queue_it/queable_by_name_spec.rb

```diff
@@ -6,18 +6,31 @@
       let(:nodable) { create(:user) }
 
       it 'expect queue to not be empty' do
-        a = create(:user)
-        b = create(:user)
+        user_a = create(:user)
+        user_b = create(:user)
+        user_c = create(:user)
 
         expect(task.queues.count).to eq(0)
 
-        task.queue("my_users").push(a)
-        task.queue("my_users").push(b)
+        task.queue("my_users").push(user_a)
 
         expect(task.queues.count).to eq(1)
+        expect(task.queue("my_users").nodes.count).to eq(1)
+        expect(task.queue("my_users").count_of_nodes).to eq(1)
```

Comment:
nodes. count and count_of_nodes are different?

Reply:
yes.
nodes is checking the nodes table
count_of_nodes is a calculated field on the queue table. it is possible that this field is out of sync if not managed correctly